docs/server-side/v2/api-reference/stream-key/create-rtmp-stream-key.mdx

```diff
@@ -1,11 +1,11 @@
 ---
 title: Create RTMP stream key
-nav: 3.169
+nav: 3.72
 ---
 
 Use this request to generate an RTMP stream key and RTMP URL for any room, enabling HLS live streaming from an external RTMP broadcaster to the specified room.
```

Comment:
Can we also add a note that stream key APIs dont work on disabled rooms and room need to be enavled (using the room APIs) before using any streamkey API?

Reply:
done